relay-profiling/src/measurements.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct Measurement {
+    // nanoseconds elapsed since the start of the profile (wall clock)
+    elapsed_since_start_ns: u64,
+    unit: String,
```

Comment:
having the unit on every measurement seems like overkill since each key in the `measurements` should have exactly 1 unit unless you're imagining a scenario where the a SDK can send a measurement like `foo` and uses different units at different times.